frame/evm/src/backend.rs

```diff
@@ -140,6 +140,10 @@ impl<'vicinity, T: Trait> ApplyBackend for Backend<'vicinity, T> {
 		I: IntoIterator<Item=(H256, H256)>,
 		L: IntoIterator<Item=evm::backend::Log>,
 	{
+		#[cfg(storage_tracing)] {
```

Comment:
I think feature conditionals should be `#[cfg(feature = "storage-tracing")]`, or does this work?